src/config/index.ts

```diff
@@ -27,11 +27,11 @@ export const CAKE_PER_BLOCK = new BigNumber(7)
 export const BLOCKS_PER_YEAR = new BigNumber((60 / BSC_BLOCK_TIME) * 60 * 24 * 365) // 10512000
 export const CAKE_PER_YEAR = CAKE_PER_BLOCK.times(BLOCKS_PER_YEAR)
 export const BASE_URL = 'https://ricefarm.fi'
-export const BASE_EXCHANGE_URL = 'https://exchange.pancakeswap.finance'
-export const BASE_ADD_LIQUIDITY_URL = `${BASE_URL}/add`
+export const BASE_EXCHANGE_URL = `${BASE_URL}/swap`
+export const BASE_ADD_LIQUIDITY_URL = `${BASE_URL}/pool`
+export const BASE_LIQUIDITY_POOL_URL = `${BASE_URL}/pools`
 export const BASE_V1_EXCHANGE_URL = 'https://teslasafe.ricefarm.fi'
-export const BASE_V1_ADD_LIQUIDITY_URL = `${BASE_V1_EXCHANGE_URL}/#/add`
-export const BASE_LIQUIDITY_POOL_URL = `${BASE_URL}/pool`
+export const BASE_V1_ADD_LIQUIDITY_URL = `https://teslasafe.ricefarm.fi/#/pool`
```

Comment:
This is incorrect as no change is necessary since v1 is still the same.

should still be `${BASE_V1_EXCHANGE_URL}/#/add`